Google Cloud/google-drive-reports/ingest/parser.yml

```diff
@@ -0,0 +1,31 @@
+name: google-drive-reports
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+
+  - name: set_ecs_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          event.kind: "event"
+          event.category: ["file"]
+          event.type: ["access"]
+
+          user.id: "{{json_event.message.id.customerId}}"
+
+          network.application: "{{json_event.message.id.applicationName}}"
+          source.ip: "{{json_event.message.ipAddress}}"
+
+      - set:
+          user.email: "{{json_event.message.actor.email}}"
+        filter: '{{ json_event.message.actor.type == "USER"}}'
+
+      - set:
+          google-drive-reports.parameters.owner_team_drive_id: '{% for param in json_event.message.events[0].parameters %}{% if param.name == "owner_team_drive_id" %}{{param.value}}{% endif %}{% endfor %}'
+          google-drive-reports.parameters.owner: '{% for param in json_event.message.events[0].parameters %}{% if param.name == "owner" %}{{param.value}}{% endif %}{% endfor %}'
```

Comment:
You can use the ecs field [file.owner](https://www.elastic.co/guide/en/ecs/8.6/ecs-file.html#field-file-owner)